packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -660,6 +707,41 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
     };
   }
 
+  private _getPage(itemIndex: number, visibleRect: IRectangle): {
+    itemCount: number;
+    height: number;
+    data?: any;
+  } {
```

Comment:
No, this interface forces two of the members (`itemCount` and `height`) to longer be optional. It's a different interface.